boards/arm/stm32f3_disco/stm32f3_disco_defconfig

```diff
@@ -21,6 +21,10 @@ CONFIG_UART_STM32_PORT_1=y
 CONFIG_CONSOLE=y
 CONFIG_UART_CONSOLE=y
 
+#enable I2C
+CONFIG_I2C=y
+CONFIG_I2C_STM32_V2=y
```

Comment:
Can't we get this in SoC config?
For instance in arch/arm/soc/st_stm32/stm32f3/Kconfig.series:
select I2C_STM32_V2 if I2C

Reply:
done